deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -1030,6 +1029,16 @@ handle_aux(leader, cast, {#return{msg_ids = MsgIds,
         _ ->
             {no_reply, Aux0, Log0}
     end;
+handle_aux(leader, _, {handle_tick, Fun, Args},
+           #?AUX{tick_pid = undefined} = Aux, Log, _) ->
+    Pid = spawn(rabbit_quorum_queue, Fun, Args),
```

Comment:
I think we can skip the monitor churn and just check the previous pid using `is_process_alive/1` on the next tick

Reply:
Done! @kjnilsson

Reply:
Actually, need to remove unnecessary function clause. Hang on....

Reply:
Aaaaand done.